inputfiles/overridingTypes.jsonc

```
@@ -1705,6 +1705,9 @@
                         "labels": {
                             "name": "labels",
                             "overrideType": "NodeListOf<HTMLLabelElement>"
+                        },
+                        "type": {
+                            "overrideType": "\"submit\" | \"reset\" | \"button\" | undefined"
```

```suggestion
                            "overrideType": "\"submit\" | \"reset\" | \"button\"
```

It cannot be `undefined` as it defaults to `submit` per the spec: https://html.spec.whatwg.org/multipage/form-elements.html#attr-button-type

>The [missing value default](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#missing-value-default) and [invalid value default](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#invalid-value-default) are the [Submit Button](https://html.spec.whatwg.org/multipage/form-elements.html#attr-button-type-submit-state) state.